packages/ember-routing/lib/services/router.js

```diff
@@ -0,0 +1,48 @@
+/**
+@module ember
+@submodule ember-routing
+*/
+
+import {
+  Service,
+  readOnly
+} from 'ember-runtime';
+import { get } from 'ember-metal';
+import RouterDSL from '../system/dsl';
+
+/*
```

Comment:
The docstrings don't seem to match out usual style, see [Router](https://github.com/emberjs/ember.js/blob/master/packages/ember-routing/lib/system/router.js#L226-L253) for example.